src/Nri/Ui/TextArea/V2.elm

```diff
@@ -211,15 +211,7 @@ calculateMinHeight textAreaStyle specifiedHeight =
                     singleLineHeight
 
         DefaultHeight ->
-            case textAreaStyle of
-                DefaultStyle ->
-                    InputStyles.textAreaHeight
-
-                WritingStyle ->
-                    InputStyles.writingMinHeight
```

Comment:
A Writing textarea is still specified to have a min-height of 150px: https://github.com/NoRedInk/noredink-ui/pull/65/files#diff-603eb374d349e2a301c6d44684f6016fR144

Should we use that 150px here as well, or was the distinction of writing vs other styles no longer needed when it's set to auto-resize?

Reply:
Good catch, thanks! Will push a fix.

Reply:
Added back the height of 150px for text areas. Also checked the styleguide app to see the min-height is applied correctly. Seems to look good!